cli/bench_command.go

```diff
@@ -215,25 +233,33 @@ func (c *benchCmd) bench(_ *kingpin.ParseContext) error {
 		// create the stream for the benchmark (and purge it)
 		nc, err := nats.Connect(opts.Config.ServerURL(), natsOpts()...)
 		if err != nil {
-			log.Fatalf("NATS connection failed: %s", err)
+			log.Fatalf("NATS connection failed %s", err)
 		}
 
 		js, err = nc.JetStream(nats.MaxWait(c.jsTimeout))
 		if err != nil {
-			log.Fatalf("Couldn't get the JetStream context: %v", err)
+			log.Fatalf("Couldn't get the JetStream context %v", err)
 		}
 		if c.kv {
+			// There is no way to purge all the keys in a KV bucket in a single operation so deleting the bucket instead
```

Comment:
Normal stream purge?

Reply:
I meant no way through the Go kv API. There is a kv.Purge but it it's different (works on a key not the whole bucket). You're right it's more efficient to purge the underlying stream, but since that assumes you know the convention to make the stream name from the bucket name, still think it would be better for nats.go to have a 'kv.PurgeAll()' (or PurgeBucket() ?) call that does the purge of the underlying stream while isolating the user to have to know that naming convention. Should I create an issue for that on nats.go?

Reply:
Yeah I think that was in the initial spec - however you can see in kv info how to get the real jetstream stream info with name and all for a traditional purge.

Doesnt matter its not like this is wrong, just a note, so dont need to change

Reply:
Right I changed it to do the purge by composing the underlying stream name directly from the subject name, and will file an issue in nats.go because I think that call is missing.